### PR TITLE
Add v2 io support

### PR DESCRIPTION
**Pull request by crosbymichael**
Signed-off-by: Michael Crosby <crosbymichael@gmail.com>

### CODE REVIEW

File: v2/manager.go [hunk 1]
```diff
@@ -245,12 +251,17 @@ func (c *Manager) Procs(recursive bool) ([]uint64, error) {
 	return processes, err
 }
 
-func (c *Manager) Stat() (map[string]uint64, error) {
+var singleValueFiles = []string{
+	"pids.current",
+	"pids.max",
```
Comment (AkihiroSuda) [inline v2/manager.go]: can be another PR, but we need `memory.current` and `memory.swap.current`?
https://facebookmicrosites.github.io/cgroup2/docs/memory-controller.html

**Comment by crosbymichael** [comment]
@Zyqsempai this has the code for how we will handle stats
